Delivered-To: raj.khem@gmail.com
Received: by 10.90.86.4 with SMTP id j4cs313304agb;
        Sat, 8 Jan 2011 16:45:19 -0800 (PST)
Received: by 10.227.155.75 with SMTP id r11mr17188266wbw.3.1294533918432;
        Sat, 08 Jan 2011 16:45:18 -0800 (PST)
Return-Path: <yann.morin.1998@anciens.enib.fr>
Received: from smtp.smtpout.orange.fr (smtp01.smtpout.orange.fr [80.12.242.123])
        by mx.google.com with ESMTP id k3si33753340wbx.29.2011.01.08.16.45.17;
        Sat, 08 Jan 2011 16:45:18 -0800 (PST)
Received-SPF: neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) client-ip=80.12.242.123;
Authentication-Results: mx.google.com; spf=neutral (google.com: 80.12.242.123 is neither permitted nor denied by best guess record for domain of yann.morin.1998@anciens.enib.fr) smtp.mail=yann.morin.1998@anciens.enib.fr
Received: from roazhon.bzh.lan ([90.32.245.227])
	by mwinf5d24 with ME
	id tClC1f0024v5z3u03ClHDf; Sun, 09 Jan 2011 01:45:17 +0100
From: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
To: uclibc@uclibc.org
Cc: Khem Raj <raj.khem@gmail.com>,
	Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>,
	Carmelo AMOROSO <carmelo.amoroso@st.com>
Subject: [PATCH 5/7] ARM: #include <bits/arm_asm.h> where __USE_BX__ is used
Date: Sun,  9 Jan 2011 01:45:08 +0100
Message-Id: <1294533910-19305-6-git-send-email-yann.morin.1998@anciens.enib.fr>
X-Mailer: git-send-email 1.7.1
In-Reply-To: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>
References: <1294533910-19305-1-git-send-email-yann.morin.1998@anciens.enib.fr>

The check for __USE_BX__ will be available in bits/arm_asm.h,
so the latter must be included wherever the former is used.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
Cc: Khem Raj <raj.khem@gmail.com>
Cc: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Cc: Carmelo AMOROSO <carmelo.amoroso@st.com>
---
 ldso/ldso/arm/dl-startup.h      |    1 +
 libc/sysdeps/linux/arm/sysdep.h |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/ldso/ldso/arm/dl-startup.h b/ldso/ldso/arm/dl-startup.h
index a95389d..2dfdaff 100644
--- a/ldso/ldso/arm/dl-startup.h
+++ b/ldso/ldso/arm/dl-startup.h
@@ -7,6 +7,7 @@
  */
 
 #include <features.h>
+#include <bits/arm_asm.h>
 
 #if !defined(__thumb__)
 __asm__(
diff --git a/libc/sysdeps/linux/arm/sysdep.h b/libc/sysdeps/linux/arm/sysdep.h
index 013f88c..e498695 100644
--- a/libc/sysdeps/linux/arm/sysdep.h
+++ b/libc/sysdeps/linux/arm/sysdep.h
@@ -21,6 +21,7 @@
 #define _LINUX_ARM_SYSDEP_H 1
 
 #include <common/sysdep.h>
+#include <bits/arm_asm.h>
 
 #include <sys/syscall.h>
 /* For Linux we can use the system call table in the header file
-- 
1.7.1

